src/routes/auth.js

```diff
@@ -30,9 +32,10 @@ router.post(
 		})
 	})
 )
-
+router.use(isAuthenticated())
 router.post(
 	'/create-user',
+	isAdmin,
```

Comment:
@itzgeebee, since this is a router with one endpoint being authenticated and the other not being authenticated. let's keep the `isAuthenticated` for the specific route for now so its clear what it applies to.

Personally, before this pr, I didn't know where the `router.use` is put matters but learnt something new . Thanks :)

something like
```suggestion
router.post(
	'/create-user',
	isAuthenticated,
	isAdmin,
```

Reply:
@jocrah so, this was how I did it at first and for some reason it didn't work. The request kept going without end till I used the router.use(). Still tried again and it's giving the same result

Reply:
> @jocrah so, this was how I did it at first and for some reason it didn't work. The request kept going without end till I used the router.use(). Still tried again and it's giving the same result

@jocrah Apparently, "isAuthenticated" has to be called for it to work well. I have updated it now.

Reply:
yea true, realized same too. was trying it locally with the correct one and it worked but my suggestion didn't capture that